repository_service_tuf_api/api/metadata.py

```diff
@@ -31,6 +31,25 @@ def post(payload: metadata.MetadataPostPayload):
     return metadata.post_metadata(payload)
 
 
+@router.put(
+    "/",
```

Comment:
I think it should be a specific endpoint "/api/v1/metadata/<name>"
Also, I would use the following behavior. 
No rolename = bump all online roles.